efi/meson.build

```diff
@@ -1,6 +1,25 @@
 generate_sbat = find_program('generate_sbat.py', native: true)
 generate_binary = find_program('generate_binary.py', native: true)
 
+# get source version, falling back to empty
+git = find_program('git', required : false)
+if git.found()
+  source_version = run_command(git, 'describe').stdout().strip()
```

Comment:
I think this needs more guarding.  What happens if you're working on a tarball in an environment with git installed?  I think this command will fail and will cause the build to fail too, no?

Reply:
Seems to work okay, using `gitXXX` as a test:

    Program gitXXX found: NO
    Configuring swid.xml using configuration
    ...
    [8/8] Generating efi/fwupdx64.efi with a custom command

Reply:
I meant having git program but no .git directory. Did you check that?

Reply:
Ahh! Gotcha, sorry. It does, but only by accident. I've added a `check: false` to be explicit about it.